src/main/java/com/gdschongik/gdsc/domain/member/api/AdminMemberController.java

```diff
@@ -35,10 +36,13 @@ public class AdminMemberController {
 
     private final AdminMemberService adminMemberService;
 
-    @Operation(summary = "전체 회원 목록 조회", description = "전체 회원 목록을 조회합니다.")
+    @Operation(summary = "회원 상태별 목록 조회", description = "정회원, 준회원, 게스트별로 조회합니다.")
```

Comment:
MemberRole도 있고 MemberStatus도 있으므로 아래와 같이 바꾸는 게 좋을 것 같습니다.
pr 제목도 같이 부탁드립니다~
```suggestion
    @Operation(summary = "회원 역할별 목록 조회", description = "정회원, 준회원, 게스트별로 조회합니다.")
```

Reply:
앗 알겠습니다~~